pages/wiki/watchfaces-creation.md

```diff
@@ -77,6 +78,31 @@ layout: documentation
     }</code></pre>
   <p>The UI slider then acts as a controller to see how the watchface reacts to the different values. Note that the slider gives a real value from 0.0 to 1.0. To simulate the integer 0 to 100 provided by the real battery on the watch, we scale and convert to a fixed value in the code above.</p>  
 </div>
+<div class="page-header">
+  <h1 id="how-to-share">Share your watchface</h1>
+</div>
+<div>
+  <p>Pull requesting your finished watchface to the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchfaces repo</a> is always very welcome! Follow this guide to ensure consitency with existing community watchfaces and a quick review process.</p>
+  <p>Edit the <code>README.md</code> and sort your watchface entry into the list alphabetically. The required thumbnails can be conveniently created by using the <code>./test-in-qmlscene.sh</code> script. </p>
+  <p>Square thumbnails are taken on the iconic flatmesh background. This helps to easily compare the watchfaces visually on the same background when scrolling through the list. Flatmesh is downloaded on start of the script if you have not placed a custom <code>background.jpg</code> to top level already.</p>
+  <p>Round thumbnails are ment to present watchfaces on a background the developer found to be most suitable. Save this custom background as <code>background-round.jpg</code> and the script will use it when generating the previews.</p>
+  <p>Pressing the <code>Generate previews</code> button saves and correctly names three high quality PNG images. Those can either be found at top level during runtime of qmlscene, in case you like to process them manually. Or you can use the images that get automatically created and copied to the correct folders by the script as soon as you close the qmlscene window.</p>
+  <p>320x320px in WEBP format at 85% compression is used for the web previews. The <code>watchfaces-preview</code> images are generated in various screen sizes to fit the watchface gallery in settings for all supported screen sizes. The format we use here is PNG at 70% compression level. Since WEBP files, though much smaller in filesize, take up to 300% longer to decompress. Which would be noticable in loading times on the watch according to our testing.</p>
```

Comment:
I don't think the implementation details are worth documenting. I can imagine the designer who made it here only wants to share his watchface and does not care so much :p

Reply:
Yupp! The 3 paragraphs where written before i added the "Wrapping up," part which additionally makes the few real info redundant. Cutting those.